google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/stub/BigtableInstanceAdminStubSettings.java

```diff
@@ -749,15 +769,15 @@ private static Builder initDefaults(Builder builder) {
                       .setInitialRpcTimeout(Duration.ZERO) // ignored
                       .setRpcTimeoutMultiplier(1.0) // ignored
                       .setMaxRpcTimeout(Duration.ZERO) // ignored
-                      .setTotalTimeout(Duration.ofMillis(300000L))
+                      .setTotalTimeout(Duration.ofMillis(600000L))
```

Comment:
Duration.ofMinutes(10)

Reply:
This code is autogenerated by gapic-generator. So it would have to change in its templates